consensus/obcpbft/events.go

```diff
@@ -258,95 +258,35 @@ func (et *eventTimerImpl) loop() {
 //
 // ------------------------------------------------------------
 
-type eventType int
-
-const (
-	workEventID eventType = iota
-	viewChangeTimerEventID
-	execDoneEventID
-	stateUpdatedEventID
-	stateUpdatingEventID
-	pbftMessageEventID
-	batchMessageEventID
-	batchTimerEventID
-	viewChangedEventID
-	complaintEventID
-	batchExecEventID
-)
-
 // workEvent is a temporary type, to inject work
 type workEvent func()
```

Comment:
Now we can move the event types from event.go to the respective users and make event.go completely oblivious to the events.